bait-files/src/main/webapp/submission.html

```diff
@@ -25,15 +26,15 @@ <h2>What do you know about this person?</h2>
         <ul>
             <li id='name-input'>
                 <label for="name">Full Name:</label><br>
-                <input type="text" id="name-input" name="name-input">
+                <input type="text" id="name-input" name="name-input" onkeydown="return /[a-z, ]/i.test(event.key)">
```

Comment:
I'd make a helper method for this regular expression check here like you did for the username one.  And call it something like keyIsValidForName.

Reply:
for this I ended up going with the checkSpace method so that I could reuse it for the name and username fields since each cannot have spaces